sharktank/sharktank/ops/default_impls.py

```diff
@@ -405,7 +405,7 @@ def rms_norm_default(x, weight, *, epsilon: float) -> Tensor:
     variance = x.pow(2).mean(-1, keepdim=True)
     output = x * elementwise(torch.rsqrt, variance + epsilon)
     # The cast here is to match the hf implementation, affects numerics
-    output = weight * to(output, weight.dtype)
+    output = unbox_tensor(weight) * to(output, weight.dtype)
```

Comment:
I don't think this breaks anything but why do we need it?

Reply:
When exporting, torch gets upset when the multiplication has one argument of a `sharktank` tensor and one of a `torch` tensor type. The `to` op always returns a torch tensor, so this fixes them both to be `torch` tensors. Almost all the functions in `default_impls.py` are calling `unbox_tensor` on their inputs. I'm not sure why this patch turned up this particular issue, but it was in fact failing before I added this.

Reply:
Actually, using that fix breaks other things. Changing it to an explicit `ops.elementwise(torch.mul, x, y)` works better.